### DISCUSSION TITLE
Is my ID valid?

### DISCUSSION THREAD

**Discussion by ThiagoAlexsander** [community/community#141087] [GitHub Education] [answered]
### Hi everyone,

I uploaded a photo of my digital student ID (since I don't have a physical one), is this considered valid proof of enrollment?

**Comment by AkalankaKaushalya** [comment]
Hmm is oky

**Comment by VeerammalV** [comment, marked as answer]
Yes, a digital student ID can be considered valid proof of enrollment for GitHub Education, as long as it clearly shows your name, photo, and enrollment status. However, it’s always a good idea to verify the specific requirements in GitHub’s guidelines to ensure your submission meets their criteria.
